VALUES.md

```diff
@@ -0,0 +1,66 @@
+# Ayo.js Community Values
+
+Ayo.js is about the humans that help make it happen. It's a project aimed at
+creating a new foundation of project governance and management that brings
+**humans** and their concerns front and center.
+
+You can pronounce it however you want: be it “ey-yo”, “I-oh”, or “Awooooo” — the
+Ayo.js project strives to create an environment where you can feel at home and
+focus on your contributions to this nice shared space that we all benefit from.
+
+## Ayo.js is more than the code in its repository.
```

Comment:
Headings don't usually have full stops.

Reply:
This was an intentional decision I made when I wrote the initial draft to make these stand out as definitive statements in addition to headings. I'm fine with changing that if it doesn't work.